### PR TITLE
[icons] feat: new icon "floating-point"

### PR DESCRIPTION
**Pull request by float-icon**
#### Checklist

- [ ] Update documentation

#### Changes proposed in this pull request:

Adds a float icon reading "1.0".

#### Screenshot

20px_float
16px_float

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/blueprint](https://github.com/palantir/blueprint), @float-icon! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: packages/icons/icons.json [hunk 1]
```diff
@@ -3883,5 +3883,12 @@
         "tags": "open, application, new-tab, external",
         "group": "interface",
         "codepoint": 62251
+    },
+    {
+        "displayName": "Float",
+        "iconName": "float",
+        "tags": "number, float, double, decimal",
+        "group": "interface",
```
Comment (adidahiya) [inline packages/icons/icons.json]: let's put it into Miscellaneous with the other (array) data types. the icons list is due for a re-organization some time; we should have a separate category for data types...

```suggestion
        "group": "miscellaneous",
```

File: packages/icons/icons.json [hunk 2]
```diff
@@ -3883,5 +3883,12 @@
         "tags": "open, application, new-tab, external",
         "group": "interface",
         "codepoint": 62251
+    },
+    {
+        "displayName": "Float",
+        "iconName": "float",
```
Comment (adidahiya) [inline packages/icons/icons.json]: I think we should be more specific and call this "floating-point". You will need to rename the SVG files to match the iconName here after you do that.

```suggestion
        "displayName": "Floating point number",
        "iconName": "floating-point",
```

**Review by adidahiya — CHANGES_REQUESTED** [review]
SVGs look good, but I'd like to change the metadata

**Comment by float-icon** [comment]
> SVGs look good, but I'd like to change the metadata

done

**Review by adidahiya — APPROVED** [review]

**Pull request merged**